pages/content-ui/script/entryPoints.ts

```diff
@@ -0,0 +1,52 @@
+import { readdirSync, existsSync, mkdirSync, writeFileSync, Dirent, readFileSync, renameSync } from 'fs';
+import { resolve } from 'node:path';
+
+const rootDir = resolve(__dirname, '..');
+const srcDir = resolve(rootDir, 'src');
+
+export const getEntryPoints = (options: { include?: string[]; exclude?: string[]; createRecursively?: boolean }) => {
+  const entryPoints: Record<string, string> = {};
+
+  const foldersToProcess =
+    options.include ||
+    readdirSync(srcDir, { withFileTypes: true })
+      .filter((item): item is Dirent => item.isDirectory())
```

Comment:
Why not like that?

```suggestion
      .filter((item: : Dirent) => item.isDirectory())
```

I don't really know this kind of typing, that's interesting

Reply:
This is an unnecessary assertion.

Reply:
> This is an unnecessary assertion.

erm we only want folder name, and not file names, readdirsync will read filenames too which we don't want.